pmd-packager/upgrade.sh

```diff
@@ -1,5 +1,7 @@
-VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | jq --raw-output ".tag_name" | sed 's:.*/::')
+VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | grep '.tag_name' | sed 's:.*/::' | sed 's:",::')
```

Comment:
Alternatively:

```
VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | jq -r .tag_name)
VERSION=${VERSION##pmd_releases/}
```

Reply:
I find that jq has to be added, but grep is built-in, so I use grep... which means I need the extra command.